mcp_extension/scan_tool_test.go

```diff
@@ -1,5 +1,5 @@
 /*
- * © 2025 Snyk Limited
+ * 2025 Snyk Limited
```

Comment:
see above, the pleasures of AI assistants messing up stuff they shouldn't touch ;)